news/models.py

```diff
@@ -11,6 +11,10 @@ class NewsArticle(models.Model):
     title = models.CharField(max_length=150)
     html_article_text = models.CharField(default="", max_length=5000)
     publication_date = models.DateTimeField()
+    image = models.ImageField(
```

Comment:
Maybe we should add a compressed image as well, for example like we do in exhibitors and events